itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/parse/TestReplicationScenariosAcrossInstances.java

```diff
@@ -1114,13 +1115,18 @@ public void testIfCkptAndSourceOfReplPropsIgnoredByReplDump() throws Throwable {
             .dump(primaryDbName, Collections.emptyList());
 
     // Incremental Repl A -> B with alters on db/table/partition
-    WarehouseInstance.Tuple tupleReplicaInc = replica.load(replicatedDbName, primaryDbName)
-            .run("repl status " + replicatedDbName)
+    replica.load(replicatedDbName, primaryDbName);
```

Comment:
- Perform bootstrap dump and load - should pass
- Perform dump on target db - should fail
- Alter target db to remove repl.target.for and perform dump on target db - should pass
- perform incremental dump and load (src->target). - should pass
- Perform dump on targetdb  - should fail.

Reply:
Done

Reply:
why not to have a separate test for the above steps?